src/api_request.py

```diff
@@ -23,7 +23,7 @@ def get_pano_location(pano_id):
     
     # WGS84 to RD conversion 
     point = ogr.Geometry(ogr.wkbPoint)
-    point.AddPoint(geom[0], geom[1])
+    point.AddPoint(max(geom), min(geom))
```

Comment:
This is ugly, but it works because the point has to be in the Netherlands anyway since we're projecting to RD NEW.